src/prefect/engine/results/s3_result.py

```diff
@@ -21,25 +19,15 @@ class S3Result(Result):
 
     Args:
```

Comment:
Docstring should also be updated like the GCSResult to mention the default secret that it can use